xray/commands/audit/audit.go

```diff
@@ -165,6 +165,7 @@ func RunAudit(auditParams *AuditParams) (results *Results, err error) {
 	if err = clientutils.ValidateMinimumVersion(clientutils.Xray, auditParams.xrayVersion, scangraph.GraphScanMinXrayVersion); err != nil {
 		return
 	}
+	results.ExtendedScanResults.XrayVersion = auditParams.xrayVersion
```

Comment:
why we need it?

Reply:
@sverdlov93 When we convert to Sarif format, the tool has an attribute of version, so we need to pass the x-ray version